components/lib/ChangeableText/ChangeableText.tsx

```diff
@@ -57,18 +57,25 @@ export const ChangeableText: FC<IChangeableTextProps> = ({ value, ...props }) =>
           className={classes.input}
         />
       </div>
-      {props.loading && (
+      {/* {props.loading && (
```

Comment:
why didn't you remove this. We do not need commented code neither in production nor in development. It is totally invalid

Reply:
Aha tshunarli